src/Pages/Home.tsx

```diff
@@ -1,17 +1,491 @@
-import NavMain from "../Components/NavMain";
 import HeroSection from "../Components/HeroSection";
-import LogoClouds from "../Components/LogoClouds";
+import AboutMe from "../Components/AboutMeSection";
+import ContactForm from "../Components/ContactForm";
+import MaxJGruberLogo from "../Assets/MaxJGruber_LOGO.png";
+import Title from "../Components/Title";
+import * as CVEn from "../Content/CVs/MaxJGruber_CV.pdf";
+import * as CVFr from "../Content/CVs/MaxJGruber_CV_FR.pdf";
+import * as certificate from "../Content/Certificate/IRONHACK_CERTIFICATE.pdf";
+
+import { Fragment, useState } from "react";
+import { Dialog, Transition, Disclosure } from "@headlessui/react";
```

Comment:
There's an actual convention for the order of imports in a file- you should first import external imports and then internal. 
This can be enforced with ESLint I believe